dot/core/mocks_test.go

```diff
@@ -1164,12 +1164,11 @@ func (mr *MockRuntimeInstanceMockRecorder) Validator() *gomock.Call {
 }
 
 // Version mocks base method.
-func (m *MockRuntimeInstance) Version() (runtime.Version, error) {
+func (m *MockRuntimeInstance) Version() runtime.Version {
```

Comment:
was not these changes already merged in another branch? Should you rebase this branch with development?

Reply:
Indeed, this is fixed up, sorry about that!